docs/sql/sql-full-text.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Full-Text Search
+parent: SQL
+nav_order: 2
+---
+
+# Full-text search
+
+Use SQL commands for full-text search. For more information about full-text queries in Elasticsearch, see [Full-text queries](../../elasticsearch/full-text/).
+
+
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Match
+
+To search for text in a single field, use `MATCHQUERY` or `MATCH_QUERY` functions.
+
+Pass in your search query and the field name that you want to search against.
+
+
+```sql
+SELECT account_number, address
+FROM accounts
+WHERE MATCH_QUERY(address, 'Holmes')
+```
+
+Or,
+
+```sql
+SELECT account_number, address
+FROM accounts
+WHERE address = MATCH_QUERY('Holmes')
+```
+
+
+| account_number | address
+:--- | :---
+1 | 880 Holmes Lane
+
+
+## Multi match
+
+To search for text in multiple fields, use `MULTI_MATCH`, `MULTIMATCH`, or `MULTIMATCHQUERY` functions.
+
+For example, to search for `Dale` in either the `firstname` or `lastname` fields:
```

Comment:
For example, search for `Dale` in either

Reply:
Done.